src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    static String getRandomBall() {
+        Random random = new Random();
+        int randomNumber = random.nextInt(101);
+
+        Ball ball = new Ball();
+        ball.setNumber(randomNumber);
+        ball.setColour(getRandomColor());
```

Comment:
Since you are extending ColorSupplier, the method call getRandomColor() does not need to be static. However, as per the first comment, the use of ColorSupplier should be through composition, not inheritance.